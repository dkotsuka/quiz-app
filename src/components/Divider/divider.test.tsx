import React from 'react'
import { Divider } from '.'
import {create} from 'react-test-renderer';

describe("Divider", () => {
    it("vertical divider matches snapshot", () => {
        const divider = create(<Divider vertical width={2}/>)
        expect(divider).toMatchSnapshot()
    })

    it("horizontal divider matches snapshot", () => {
        const divider = create(<Divider vertical={false} width={2}/>)
        expect(divider).toMatchSnapshot()
    })
})