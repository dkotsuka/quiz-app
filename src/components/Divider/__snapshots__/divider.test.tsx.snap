// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider horizontal divider matches snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "borderColor": "#B8BED5",
        "borderTopWidth": 2,
        "width": "100%",
      },
    ]
  }
  vertical={false}
  width={2}
/>
`;

exports[`Divider vertical divider matches snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "borderColor": "#B8BED5",
        "borderLeftWidth": 2,
        "height": "100%",
      },
    ]
  }
  vertical={true}
  width={2}
/>
`;
